lib/virt.py

```diff
@@ -819,6 +827,51 @@ def ssh_keygen(path):
     subprocess.run(['ssh-keygen', '-N', '', '-f', path], stdout=DEVNULL, check=True)
 
 
+def translate_ssg_kickstart(profile):
+    """
+    Parse (and tweak) a kickstart shipped with the upstream content
+    into class Kickstart instance.
+    """
+    ks_text = ''
+    with open(util.get_kickstart(profile)) as f:
+        for line in f:
+            line = line.rstrip('\n')
+
+            # use our own password
+            if re.match('^rootpw ', line):
+                line = f'rootpw {GUEST_LOGIN_PASS}'
+
+            # don't hardcode interface name
+            if re.match('^network ', line):
+                line = re.sub(' --device[^ ]+', '', line)
+
+            # shrink some unnecessary partitions
+            if re.match('^logvol / ', line):
+                line = re.sub('--size=[^ ]+', '--size=4000', line)
```

Comment:
Maybe I should check whether the original isn't `>4000` to make sure we're not enlarging it and going over the limit.

Reply:
Dunno, if there is single value we could use for all profiles, as some have `/` <4000 - ANSSI https://github.com/ComplianceAsCode/content/blob/master/products/rhel8/kickstart/ssg-rhel8-anssi_bp28_high-ks.cfg#L99
and some have it `>>4000` as OSPP https://github.com/ComplianceAsCode/content/blob/master/products/rhel8/kickstart/ssg-rhel8-ospp-ks.cfg#L96